main_routers/characters_router.py

```
@@ -1834,6 +1838,18 @@ async def voice_clone(file: UploadFile = File(...), prefix: str = Form(...), ref
     
     if is_local_tts:
         # ==================== 本地 TTS 注册流程 ====================
+        # MD5 去重：检查是否已有相同音频注册过的音色
```

Ruff (0.15.4)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)